Src/Plugins/Security/SEC1003.ReviewPotentiallySensitiveFiles.json

```diff
@@ -4,16 +4,111 @@
       "Description": "Review potentially sensitive files in versioned engineering content and other locations for security risk.",
       "Message": "'{0:scanTarget}' is {1:fileKind} that may compromise security if visible to attackers.",
-
       "MatchExpressions": [
+        {
+          "SubId": "1PasswordFile",
+          "FileNameAllowRegex": "(?i)\\.agilekeychain$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a 1Password manager database file" }
+        },
         {
           "SubId": "ASCIIArmoredFile",
-          "NameAllowList": "(?i)\\.asc$",
+          "FileNameAllowRegex": "(?i)\\.asc$",
           "ContentsRegex": ".",
           "MessageArguments": { "fileKind": "an ASCII armored file" }
+        },
+        {
+          "SubId": "MicrosoftBitLockerRecoveryKeyFile",
+          "FileNameAllowRegex": "(?i)\\.bek$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Microsoft BitLocker recovery key file" }
+        },
+        {
+          "SubId": "WindowsBitlockerDataFile",
+          "FileNameAllowRegex": "(?i)\\.fve$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Windows BitLocker full volume encrypted data file" }
+        },
+        {
+          "SubId": "JavaKeystoreFile",
+          "FileNameAllowRegex": "(?i)\\.jks$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Java keystore file" }
+        },
+        {
+          "SubId": "KDEWalletManagerDatabaseFile",
+          "FileNameAllowRegex": "(?i)\\.kwallet$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a KDE Wallet Manager database file" }
+        },
+        {
+          "SubId": "MicrosoftSQLDatabaseFile",
+          "FileNameAllowRegex": "(?i)\\.mdf$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a Microsoft SQL database file" }
+        },
+        {
+          "SubId": "OpenVPNClientConfigurationFile",
+          "FileNameAllowRegex": "(?i)\\.ovpn$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "an OpenVPN client configuration file" }
+        },
+        {
+          "SubId": "P12CryptographicKeyFile",
+          "FileNameAllowRegex": "(?i)\\.p12$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic key bundle" }
+        },
+        {
+          "SubId": "PEMCryptographicPrivateKeyFile",
+          "FileNameAllowRegex": "(?i)\\.pem$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic private key" }
+        },
+        {
+          "SubId": "PFXCryptographicKeyFile",
+          "FileNameAllowRegex": "(?i)\\.pfx$",
+          "ContentsRegex": ".",
+          "MessageArguments": { "fileKind": "a cryptographic key bundle" }
+        },
+        {
+          "SubId": "PKCS12CryptographicKeyFile",
```

Comment:
>PKCS12CryptographicKeyFile [](start = 20, length = 26)

btw - this is probably the right SubId for all these PKCS12 container extensions when you create the roll-up rule.